app.py

```diff
@@ -19,7 +19,13 @@ def strategy_1(x: int, y: int,
     :param att_dy: attacker y delta
     :return: dx, dy that the defender need to move
     """
-    dx, dy = -att_dx, -att_dy
+    new_x_att_pos = att_x - att_dx
+    new_y_att_pos = att_y - att_dy
```

Comment:
This is a wrong calculation for new position.

Reply:
To demonstrate:
![image](https://user-images.githubusercontent.com/1958978/30124943-b581bfd8-92eb-11e7-96c7-7f7c93c770ae.png)